Add notes about double quotes in line protocol for tag and field values (#3044)
influxdata/docs-v2: content/influxdb/v1.8/write_protocols/line_protocol_tutorial.md

weather,location=us-midwest,season=summer temperature=82 1465839830100400200
```

> When it comes to quoting the tag sets, line protocol supports single and double quotes as described in the following table:

Reviewer: ```suggestion
When using quotes in tag sets, line protocol supports single and double quotes as described in the following table:
```
